converter.go

```diff
@@ -1029,27 +1042,45 @@ func arrowToRecord(ctx context.Context, record arrow.Record, pool memory.Allocat
 			}
 			defer newCol.Release()
 		case timestampNtzType, timestampLtzType, timestampTzType:
-			if useOriginalTimestamp {
+			if arrowBatchesTimestampOption == UseOriginalTimestamp {
 				// do nothing - return timestamp as is
 			} else {
+				var unit arrow.TimeUnit
+				switch arrowBatchesTimestampOption {
+				case UseMicrosecondTimestamp:
```

Comment:
Shouldn't we add `UseSecondTimestamp` here?

Reply:
all fixed.